types/bool.go

```diff
@@ -79,14 +79,19 @@ func (b Bool) Equal(other attr.Value) bool {
 	return b.Value == o.Value
 }
 
+// IsNull returns true if the Bool represents null value.
```

Comment:
Nit: Missing article here and other IsNull

```suggestion
// IsNull returns true if the Bool represents a null value.
```